<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="menu_btn_home">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="menu_btn_modulos">
      <property name="text">
       <string>Modulos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="menu_btn_aulas">
      <property name="text">
       <string>Aulas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="menu_btn_att_dados">
      <property name="text">
       <string>Atualizar dados</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="menu_btn_att_senha">
      <property name="text">
       <string>Atualizar senha</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="menu_stacked">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>1241</width>
     <height>751</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="menu_home">
    <widget class="QLabel" name="home_label_bem_vindo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>971</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menu_modulos">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>311</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="modulos_layout_modulo_1"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>311</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="modulos_layout_modulo_2"/>
    </widget>
   </widget>
   <widget class="QWidget" name="modulos_modulo">
    <widget class="QLabel" name="modulo_label_modulo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>411</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Aluno matriculados</string>
     </property>
    </widget>
    <widget class="QListWidget" name="modulo_output_alunos_matriculados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>391</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menu_aulas">
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>140</y>
       <width>271</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="aulas_layout_aula_pronta"/>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Suas aulas prontas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aulas_btn_criar_aula">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>Criar nova aula</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aulas_btn_vincula_aula_modulo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>vincular aula a modulo</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aulas_criar_aula">
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Título da aula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="criar_aula_input_titulo_aula">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="criar_aula_input_nivel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nível da aula</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Conteúdo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="criar_aula_btn_criar_aula">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>550</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Criar aula</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="criar_aula_input_conteudo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="criar_aula_btn_escolher_video">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolher vídeo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="criar_aula_input_video">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="criar_aula_input_conteudo_download">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="criar_aula_btn_escolher_conteudo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Conteudo adicional</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aulas_vincula_aula_modulo">
    <widget class="QListWidget" name="vincula_aula_modulo_input_aula">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="vincula_aula_modulo_input_modulo">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="vincula_aula_modulo">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>480</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Vincula</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Aulas disponíveis</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>170</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Módulos possíveis</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aulas_aula">
    <widget class="QLabel" name="aula_label_aula">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>321</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="aula_label_conteudo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>321</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menu_att_dados">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>241</width>
       <height>226</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Rua</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_rua"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Número</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_numero"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bairro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_bairro"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CEP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_cep"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_cidade"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Estado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="att_dados_input_estado"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tipo endereço</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="att_dados_input_tipo_end"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Endereço</string>
     </property>
    </widget>
    <widget class="QPushButton" name="att_dados_btn_att_dados">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>480</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Atualizar dados</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>100</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dados pessoais</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>170</y>
       <width>391</width>
       <height>193</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sobrenome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_sobrenome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Data de nascimento</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_data_nasc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Celular</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="att_dados_input_celular"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Gênero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="att_dados_input_genero"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Estado civil</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="att_dados_input_estado_civil"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="menu_att_senha">
    <widget class="QPushButton" name="att_senha_btn_visualizar_senha">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>180</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualizar senha</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>266</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Senha antiga</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="att_senha_input_senha_antiga">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="att_senha_btn_att_senha">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar senha</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>266</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nova senha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="att_senha_input_verifica_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>266</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Novamente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="att_senha_input_verifica_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
